config/environment.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
+
+module.exports = function (/* environment, appConfig */) {
+  var ENV = {
+    podModulePrefix: 'ember-frost-icons/pods'
```

Comment:
Ah, yes, I missed pulling that out when I removed the pods. Good catch. Fixing now.